less/variables.less

```diff
@@ -11,4 +11,19 @@
 @silver: #C0C0C0;
 @shark: #212529;
 @eastern-blue: #17A2B8;
-@sandy-beach: #FFEBCD;
\ No newline at end of file
+@sandy-beach: #FFEBCD;
+
+@media (max-width:500px){
```

Comment:
Is there any reason why you have a media query in the `variables.less`? Were you trying to make a escaping value for `@mobile`?